### PR TITLE
import and export exception handling decorated and refactor

### PR DESCRIPTION
**Pull request by Ashutosh619-sudo**

### CODE REVIEW

File: apps/mappings/exceptions.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import logging
+import traceback
+
+
+from xerosdk.exceptions import WrongParamsError as XeroWrongParamsError, InvalidTokenError as XeroInvalidTokenError, UnsuccessfulAuthentication, InvalidGrant
```
Comment (ruuushhh) [inline apps/mappings/exceptions.py:5]: Fix indentation

File: apps/mappings/exceptions.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            error = {
+                'task': task_name,
+                'workspace_id': workspace_id,
+                'alert': False,
+                'message': None,
+                'response': None
+            }
+
+            try:
+                return func(workspace_id,*args)
```
Comment (ruuushhh) [inline apps/mappings/exceptions.py:24]: add space before `*args`

File: apps/mappings/exceptions.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            except InvalidTokenError:
+                error['message'] = 'Invalid Fyle refresh token'
+            
+            except XeroCredentials.DoesNotExist:
+                error['message'] = 'Xero Credentials not found'
+            
+            except WrongParamsError as e:
```
Comment (ruuushhh) [inline apps/mappings/exceptions.py:31]: let call it exception instead of e

File: apps/mappings/tasks.py [hunk 4]
```diff
@@ -14,6 +14,7 @@
 from fyle.platform.exceptions import WrongParamsError, InternalServerError, InvalidTokenError as FyleInvalidTokenError, PlatformError
 
 from xerosdk.exceptions import UnsuccessfulAuthentication, InvalidGrant
```
Comment (ruuushhh) [inline apps/mappings/tasks.py:16]: if we are not using these please remove

File: apps/mappings/tasks.py [hunk 5]
```diff
@@ -182,89 +183,41 @@ def upload_categories_to_fyle(workspace_id):
 
     return xero_attributes
 
+
+@handle_import_exceptions('auto_create_category_mappings')
```
Comment (ruuushhh) [inline apps/mappings/tasks.py:187]: `@handle_import_exceptions(task_name='auto_create_category_mappings')`
lets follow this format everywhere

File: apps/exceptions.py [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+
+            except InvalidTokenError as exception:
+                logger.info('Xero token expired workspace_id - %s %s', kwargs['workspace_id'], {
+                            'error': exception.response})
+                return Response(
+                    data={
+                        'message': 'Xero token expired workspace_id'
+                    },
+                    status=status.HTTP_400_BAD_REQUEST
+                )
```
Comment (ashwin1111) [inline apps/exceptions.py:63]: let's club both these exceptions

File: apps/mappings/tasks.py [hunk 7]
```diff
@@ -182,89 +179,41 @@ def upload_categories_to_fyle(workspace_id):
 
     return xero_attributes
 
+
+@handle_import_exceptions(task_name='auto_create_category_mappings')
```
Comment (ashwin1111) [inline apps/mappings/tasks.py:183]: let's send human readable format for task_name everwhere

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**